server/app/services/export/CsvExporterService.java

```diff
@@ -190,7 +190,12 @@ private String exportCsv(
                   ? applicantService.getApplicationEligibilityStatus(application, programDefinition)
                   : Optional.empty();
 
-          csvExporter.exportRecord(application, roApplicantService, optionalEligibilityStatus);
+          csvExporter.exportRecord(
+              application,
+              roApplicantService,
+              optionalEligibilityStatus,
+              programDefinition.adminName(),
+              programDefinition.adminDescription());
```

Comment:
both these fields come from programDefinition. We can pull it from the above function. instead of application.getProgram().getProgramDefinition()

Reply:
done!